integration-tests/change-directory/src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+# The property is now relative to the root project (quarkus-openapi-generator-parent) to allow CI process.
```

Comment:
hmm I think this hides a behavior we do not want, right? Since the base path should be the module, not the parent project.

Reply:
i understand your concern, although I think it will only be problematic in this test suite and not in a production environment. As long as we explain it properly on the README, it should be pretty simple to set the value properly.

Reply:
i managed to find a way to get the correct path relative to the module, hope you like it :)